packages/babel-generator/src/generators/typescript.js

```diff
@@ -348,7 +348,7 @@ export function TSTypeAliasDeclaration(node) {
   this.space();
   this.token("=");
   this.space();
-  this.print(typeAnnotation, node);
+  this.print(typeAnnotation.typeAnnotation, node);
```

Comment:
This doesn't seem right to me. If you have to drill into the child node to get it to print properly, doesn't that mean the child node isn't representing the right thing?